docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -63,26 +79,136 @@ Using a custom namespace solves problems later on because you do not have to ove
+   mimir-compactor-0                           1/1     Running     0          2m44s
+   mimir-ingester-zone-a-0                     1/1     Running     0          2m43s
+   mimir-store-gateway-zone-a-0                1/1     Running     0          2m44s
+   mimir-ingester-zone-c-0                     1/1     Running     0          2m44s
+   ```
+
+1. Wait until all of the pods have a status of `Running` or `Completed`, which might take a few minutes.
+
+## Generate some metrics for testing
+
+The Grafana Mimir Helm chart can collect metrics or logs, or both, about Grafana Mimir itself. This is called _metamonitoring_.
+In the example that follows, metamonitoring scrapes metrics about Grafana Mimir itself, and then writes those metrics to the same Grafana Mimir instance.
 
-   b. Add the following configuration to the file:
+1. Create a YAML file called `custom.yaml` for your Helm values.
+
+1. To enable metamonitoring in Grafana Mimir, add the following YAML snippet to your Grafana Mimir `custom.yaml` file:
+
+   ```yaml
+   metaMonitoring:
+     serviceMonitor:
+       enabled: true
+     grafanaAgent:
+       enabled: true
+       installOperator: true
+       metrics:
+         additionalRemoteWriteConfigs:
+           - url: "http://mimir-nginx.mimir-test.svc:80/api/v1/push"
+   ```
+
+   > **Note:** In a production environment the `url` above would point to an external system, independent of your Grafana Mimir instance, such as an instance of Grafana Cloud Metrics.
+
+1. Upgrade Grafana Mimir by using the `helm` command:
+
+   ```bash
+   helm -n mimir-test upgrade mimir grafana/mimir-distributed -f custom.yaml
+   ```
+
+## Start Grafana in Kubernetes and query metrics
+
+1. Install Grafana in the same Kubernetes cluster.
+
+   For details, see [Deploy Grafana on Kubernetes](/docs/grafana/latest/setup-grafana/installation/kubernetes/).
+
+1. If you haven't done it as part of the previous step, port-forward Grafana to `localhost`, by using the `kubectl` command:
+
+   ```bash
+   kubectl port-forward service/grafana 3000:3000
+   ```
+
+1. In a browser, go to the Grafana server at [http://localhost:3000](http://localhost:3000).
+1. Sign in using the default username `admin` and password `admin`.
+1. On the left-hand side, go to **Configuration** > **Data sources**.
+1. Configure a new Prometheus data source to query the local Grafana Mimir server, by using the following settings:
+
+   | Field | Value                                           |
+   | ----- | ----------------------------------------------- |
+   | Name  | Mimir                                           |
+   | URL   | http://mimir-nginx.mimir-test.svc:80/prometheus |
+
+   To add a data source, see [Add a data source](/docs/grafana/latest/datasources/add-a-data-source/).
+
+1. Verify success:
+
+   You should be able to query metrics in [Grafana Explore](/docs/grafana/latest/explore/),
+   as well as create dashboard panels by using your newly configured `Mimir` data source.
```

Comment:
WDYT about giving an example query? Maybe the distributor ingested samples per second? 

```
sum by (job) (rate(cortex_distributor_samples_in_total[5m]))
```

Reply:
I had that in an early version but someone commented that it can drift apart from code and not work later...